pkg/windows/windows.go

```diff
@@ -233,9 +233,10 @@ type Windows interface {
 	// ConfigureWICD ensures that the Windows Instance Config Daemon is running on the node
 	ConfigureWICD(string, string, *Authentication) error
 	// Deconfigure removes all files and networks created by WMCO and runs the WICD cleanup command.
-	Deconfigure(string, string) error
+	Deconfigure(string, string, *Authentication) error
 	// RunWICDCleanup runs the WICD cleanup command that ensures all services configured by WICD are stopped.
-	RunWICDCleanup(string, string) error
+	// Stops the WICD service prior to executing cleanup, if needed.
```

Comment:
`RunWICDCleanup ensures the WICD service  is stopped if needed and runs the cleanup command that ensures WICD managed services are stopped`

Reply:
updated